bin/migrate-org-registry-access.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+"""
+Migrate existing organizations to add default value for
+registry_access field in schema defined in organization.yaml
+
+Usage: ckanapi dump organizations --all -c $CONFIG_INI | python
+migrate-org-registry-access.py --output outfile.jsonl """
```

Comment:
`jq` is great for this kind of thing. We could replace this whole script with:
```suggestion
Usage: ckanapi dump organizations --all -c $CONFIG_INI | jq -c '.+{"registry_access":"goc"}' > outfile.jsonl """
```

Reply:
in that case, we don't need a script. I can remove it and add this command in the ticket as an important step when deploying in production?